web/l10n/en.json

```diff
@@ -26,6 +26,10 @@
     "sharedMute": "Mute",
     "sharedType": "Type",
     "sharedDistance": "Distance",
+    "sharedDayAbbreviation": "d",
```

Comment:
ExtJS probably has translation for it already. If we use their formatted, we won't need it here anymore.